ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -99,6 +150,10 @@ class LocalAssemblerData : public ProcessLib::LocalAssemblerInterface<GlobalMatr
     NodalVectorType _localRhs;
 
     unsigned const _integration_order;
+
+    std::vector<std::vector<double>> _darcy_velocities
```

Comment:
Is it possible to use `std::array<std::array<double, ShapeFunction::NPOINTS>, GlobalDim>`?